fast/stages/02-networking-nva/variables.tf

```diff
@@ -121,13 +121,17 @@ variable "psa_ranges" {
   description = "IP ranges used for Private Service Access (e.g. CloudSQL)."
   type        = map(map(string))
   default = {
-    prod = {
-      cloudsql-mysql     = "10.128.94.0/24"
-      cloudsql-sqlserver = "10.128.95.0/24"
-    }
     dev = {
-      cloudsql-mysql     = "10.128.62.0/24"
-      cloudsql-sqlserver = "10.128.63.0/24"
+      cloudsql-mysql-ew1     = "10.128.157.0/24"
```

Comment:
Per my comment, use the secondo to last and last /24 available on the carved IP estate.